### PR TITLE
Fix download servlet for quarkus deployment

### PR DESCRIPTION
**Pull request by carlesarnal**

### CODE REVIEW

File: front-end/servlet/src/main/java/io/apicurio/studio/fe/servlet/servlets/JwtDownloadServlet.java [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.Date;
+
+/**
+ * Servlet used to download content (e.g. an API design).
+ * @author eric.wittmann@gmail.com
```
Comment (EricWittmann) [inline front-end/servlet/src/main/java/io/apicurio/studio/fe/servlet/servlets/JwtDownloadServlet.java:52]: I love when I get credit for stuff I didn't write.  I'm definitely going to show this to Paolo during my performance review.  ;)

File: front-end/servlet/src/main/java/io/apicurio/studio/fe/servlet/servlets/JwtDownloadServlet.java [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+ */
+public class JwtDownloadServlet extends HttpServlet {
```
Comment (EricWittmann) [inline front-end/servlet/src/main/java/io/apicurio/studio/fe/servlet/servlets/JwtDownloadServlet.java:54]: Can we share a bunch of code between the JWT download servlet and the KC download servlet?

File: front-end/servlet/src/main/java/io/apicurio/studio/fe/servlet/servlets/JwtDownloadServlet.java (hunk 1 above)
Comment (carlesarnal) [reply front-end/servlet/src/main/java/io/apicurio/studio/fe/servlet/servlets/JwtDownloadServlet.java:52]: Well, it’s quite the same content, so you can get all the credit :).

File: front-end/servlet/src/main/java/io/apicurio/studio/fe/servlet/servlets/JwtDownloadServlet.java (hunk 2 above)
Comment (carlesarnal) [reply front-end/servlet/src/main/java/io/apicurio/studio/fe/servlet/servlets/JwtDownloadServlet.java:54]: Sure, I’ve tried to do as less changes as possible, but is the best to do.

**Pull request merged**